painttiming.bs

```diff
@@ -62,9 +87,34 @@ Formally, we consider the user agent to have "rendered" a document when it has p
 
     NOTE: The rendering pipeline is very complex, and the timestamp should be the latest timestamp the user agent is able to note in this pipeline (best effort). Typically the time at which the frame is submitted to the OS for display is recommended for this API.
 
+<dfn>Element or pseudo-element</dfn>: A [=document-sourced element=] or a [=generated content pseudo-element=].
+<dfn>Contentful</dfn>: An [=element or pseudo-element=] counts as contentful when one or more of the following apply:
+* The element has a [=text node=] child, with at least one character, excluding [=document white space characters=].
+* The element is a [=replaced image=] representing a [=loaded=] image.
+* The element's [=computed=] style contains a [=loaded=] [=background-image=].
```

Comment:
We should probably use [used value](https://www.w3.org/TR/CSS22/cascade.html#used-value) instead of computed value. By the way, computed style isn't really a spec concept it's a phrase a lot of people use informally.